### PR TITLE
Fix "possible track configurations". Fixes #3564.

### LINKED ISSUE #3564: Suspended Swinging Coaster crashes on selecting rolling/banking track
**OS:** Windows 7
**Version:** 0.0.5 Develop
**Commit:** 4298f41

Explanation of the issue:
When you try to switch the rolling/banking while building a custom Suspended Swinging Coaster, the game crashes. Upon further inspection, this is not reproducible in RCT2 Vanilla...because Suspended Swinging Coasters cannot be rolled/banked.
**The option to do so should not be there in the first place.**

Reproducible in RCT2 (vanilla)? No.

**Steps to reproduce:**
1. Start building a custom Suspended Swinging Coaster
2. Try rolling/banking the track

**Dump/Save file**

[OpenRCT2Crash.zip](https://github.com/OpenRCT2/OpenRCT2/files/258277/OpenRCT2Crash.zip)

### PR DESCRIPTION
**Pull request by Wirlie**
This bug is caused by aa929e1 because `RideTypePossibleTrackConfigurations` is used instead of the RCT2 adress `0x0097C468` and `0x0097C5D4`, for this reason, I have fixed `RideTypePossibleTrackConfigurations` and it seems to work properly.

Please review and test my changes, I'm not sure if this is the correct solution, if you have another solution please discard my changes :P

**Important**: Saved games with this problem are still having the problem (probably caused because the researched track are saved in each save game).

### CODE REVIEW

**Comment by IntelOrca** [comment]
Looks like I made a mistake converting that table from the address. Can you explain what you have changed in more detail, like what flags you changed to something else?

**Comment by Wirlie** [comment]
I only have removed the wrong flags from the rides that shouldn't have these tracks, for example:

Steeplechase shouldn't have the flag TRACK_FLAT_ROLL_BANKING because steeplechase cannot be banked.
Mini Roller Coaster and Giga Coaster shouldn't have the flag TRACK_VERTICAL_LOOP because these coasters not have this track.
etc.

I used the stable version 0.0.4 for compare the possible track of each ride.
In short ... your table is correct, only had more flags that I have removed. (If you need a table of all flags that I have removed I can do it)

I hope that you understand me, I try to write in English as best as possible :s

**Comment by IntelOrca** [comment]
Do you know if there are any flags missing from any of the ride types?

**Comment by Wirlie** [comment]
Not that I know.
